tasks/node_update.json

```diff
@@ -11,9 +10,10 @@
       "description": "Role of the node",
       "type": "Optional[Enum['manager', 'worker']]"
     },
-    "node": {
-      "description": "Hostname or ID of the node in the swarm",
+    "nodeid": {
```

Comment:
please do preserve the old name (print a message and redirect to the new parameter), so that this change does not break backwards compatibility.